src/sidebar/components/ThreadCard.tsx

```diff
@@ -36,15 +36,18 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
     },
     [frameSync]
   );
+  const threadRef = useRef<HTMLElement | null>(null);
 
   /**
-   * Is the target's event an <a> or <button> element, or does it have
-   * either as an ancestor?
+   * Is the target's event an <a> or a <button> other than the thread itself, or
+   * does it have either as an ancestor?
    *
    * @param {Element} target
    */
-  const isFromButtonOrLink = (target: Element) => {
-    return !!target.closest('button') || !!target.closest('a');
+  const isFromChildButtonOrLink = (target: Element) => {
+    return (
+      target.closest('button') !== threadRef.current || !!target.closest('a')
```

Comment:
The whole card is now wrapped in a button, so we now check if it's a button other than the one wrapped in `threadRef`.